src/registrar/templates/domain_detail.html

```diff
@@ -17,7 +17,9 @@
           <span class="text-bold text-primary-darker">
             Status:
           </span>
-            {% if domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED%}
+            {% if domain.is_expired %}
+              Expired
+            {% elif domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED%}
```

Comment:
It seems like this branch is based off of the expiry PR? Or merged that content in?

Reply:
yes, it was the intent not to have this reviewed until 1352 was merged